app/assets/javascripts/teleconsultation-fields.js

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function () {
+    $("#facility_enable_teleconsultation").click(function () {
```

Comment:
I would set some data attribute on the checkbox that we can use to find this. It may be identical to the ID, but it's a good idea to use something we control, rather than an autogenerated Rails ID.

Reply:
This ID is being set explicitly in the form, wouldn't that suffice?

Reply:
Oh I completely missed that! Perf.